larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -318,18 +369,23 @@ TEST_P(BConv2DOpTest, SimpleTest) {
   const int stride_width = param.stride_width;
   const int dilation_height_factor = param.dilation_height_factor;
   const int dilation_width_factor = param.dilation_width_factor;
-  // TOOD: currently we are ignoring padding
   const Padding padding = param.padding;
   const int num_threads = param.num_threads;
 
+  const Padding builtin_padding =
+      (padding == Padding_ONE ? Padding_VALID : padding);
+  const Padding bconv_padding =
+      (padding == Padding_ONE ? Padding_SAME : padding);
+  const int pad_values = (padding == Padding_ONE ? 1 : 0);
```

Comment:
I don't understand why we still need to `Padding_ONE` here!

Reply:
We want to test all three cases, so this `Padding_ONE` is simply used to be able to "set up the test" so to speak. It won't be passed to any of the ops. Thats why we directly define `bconv_padding` and `builtin_padding` which is what is passed to the op.

Another option is that we add another testing-parameter called `pad_value` but then it will also be done in combination with `VALID` so then it runs way more unit tests than whats needed, and it doesnt make so much sense.
(And also I think we're already using the maximum amount of testing parameters).